exporters/jaeger/internal/third_party/thrift/lib/go/thrift/multiplexed_protocol.go

```diff
@@ -160,15 +160,15 @@ func (t *TMultiplexedProcessor) ProcessorMap() map[string]TProcessorFunction {
 // the given ProcessorName or if all that is given is the FunctionName and there
 // is no DefaultProcessor set.
 func (t *TMultiplexedProcessor) AddToProcessorMap(name string, processorFunc TProcessorFunction) {
-	components := strings.SplitN(name, MULTIPLEXED_SEPARATOR, 2)
-	if len(components) != 2 {
-		if t.DefaultProcessor != nil && len(components) == 1 {
-			t.DefaultProcessor.AddToProcessorMap(components[0], processorFunc)
+	component0, component1, found := strings.Cut(name, MULTIPLEXED_SEPARATOR)
```

Comment:
Why not assigning the "proper names" here?

```suggestion
	processorName, funcName, found := strings.Cut(name, MULTIPLEXED_SEPARATOR)
```